src/angular-app/bellows/shared/audio-recorder/audio-recorder.component.ts

```diff
@@ -1,163 +1,126 @@
+import { webmFixDuration } from 'webm-fix-duration';
 import * as angular from 'angular';
-import * as lamejs from 'lamejs';
 
-declare var webkitAudioContext: {
-  new(): AudioContext;
-};
+declare var MediaRecorder: any;
```

Comment:
I'm not sure about this - is this a way to declare the type MediaRecorder?  Maybe there is a type for MediaRecorder that we can download on npm?

Reply:
Yes, Besides the fact that I two mediaRecorder vars (just removed this one, which was not used), I just installed the @types/dom-mediacapture-record devDependency to add the MediaRecorder type.